src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.wasm.cs

```diff
@@ -127,16 +127,18 @@ internal ScrollBarVisibility HorizontalScrollBarVisibility
 			}
 		}
 
+		private bool _canHorizontallyScroll;
 		public bool CanHorizontallyScroll
```

Comment:
you're setting this for FlipView, but seemingly not anywhere else. You need to set it somehow for the regular (non-FlipView) context.